base/graveyard_body.hpp

```diff
@@ -0,0 +1,21 @@
+
+#pragma once
+
+#include "base/graveyard.hpp"
+
+namespace principia {
+namespace base {
+
+template<typename T>
+Graveyard<T>::Graveyard(std::int64_t const number_of_threads)
+    : gravedigger_(number_of_threads) {}
+
+template<typename T>
+void Graveyard<T>::Bury(std::unique_ptr<T> t) {
+  gravedigger_.Add([coffin = t.release()]() {
```

Comment:
Add a TODO so that we remember to investigate the possibility of a mutable λ with `std::packaged_task` in the `ThreadPool` instead of `std::function`.

Reply:
Done.